### DISCUSSION TITLE
Context menus don't work well when scrolling overflow-y

### DISCUSSION THREAD

**Discussion by vinhphanybn1** [xyflow/xyflow#4858] [Q&A] [answered]
I tried the Menu context but I found a problem. When my screen has a scrolling overflow menu displayed in the wrong position.
But when I scroll it down to see the full reactflow view, it works fine
This is stackbliz link: 
    https://stackblitz.com/edit/kzr9rm?file=App.tsx
    
You can see my video    
https://github.com/user-attachments/assets/9c532636-b6a6-4bb4-97c8-19514bfe1ea3

**Comment by bcakmakoglu** [comment, marked as answer]
Should be fixable by subtracting `pane.x` and `pane.y` off of clientX/Y

```ts
const onNodeContextMenu = useCallback(
  (event, node) => {
    // Prevent native context menu from showing
    event.preventDefault();

    // Calculate position of the context menu. We want to make sure it
    // doesn't get positioned off-screen.
    const pane = ref.current.getBoundingClientRect();
    setMenu({
      id: node.id,
      top: event.clientY - pane.y,
      left: event.clientX - pane.x,
    });
  },
  [setMenu]
);
```

[Here's an example.](https://stackblitz.com/edit/kzr9rm-qrmafz?file=App.tsx)

**Reply by vinhphanybn1** [reply]
I checked all example, 
We always have to set width and height(
#app {
  width: 100vw;
  height: 100vh;
}).
 Is there a way to set height to automatically display all Nodes? We can disable zoom.
